db/schema.rb

```diff
@@ -46,9 +46,11 @@
     t.integer "complied_with_reentry", default: 0
     t.integer "anyone_tribe", default: 0
     t.text "tribe_details"
-    t.integer "has_quest_card", default: 0
     t.integer "anyone_stopped_work", default: 0
     t.text "stopped_work_details"
+    t.integer "has_quest_card", default: 0
+    t.text "cama_details"
```

Comment:
this `cama_details` looks like it got added?

Reply:
I've removed the stray field.

Reply:
Well now I rebased against master, so the field is back, but added by its proper commit, not this one.